src/icons/coins.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import injectSheet, { StyleSheet, StyledComponentProps } from 'react-jss';
+
+import { SvgIcon } from '@material-ui/core';
+import { SvgIconProps } from '@material-ui/core/SvgIcon';
+import DefaultLinearGradient from './_defaultLinearGradient';
+
+const styles: StyleSheet = {
+  root: {
+      transition: 'all 0.2s ease-out',
+      '&:hover': {
+          fill: 'white',
+      },
+  }
+};
+
+export const CoinsIcon = injectSheet(styles)<SvgIconProps>((props: SvgIconProps & StyledComponentProps)=> (
+  <SvgIcon style={{ width: props.width || '62px', height: props.height || '64px', fill: props.fill || 'url(#dlg)' }} {...props} viewBox={props.viewBox || "0 0 62 64"}>
```

Comment:
Shouldn't the Linear Gradient be also an prop option for the icon?
We may want the icon *without* the linear gradient. (?)

Reply:
We can use the icon without Linear Gradient but this is the default icon color `url(#dlg)`. dlg = Default Linear Gradient.
So, if we want to show the icon in another color, just need to add `fill: #colorcode` option.